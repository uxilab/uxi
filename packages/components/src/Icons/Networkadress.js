import React from 'react';
import PropTypes from 'prop-types';
import SvgIcon from '../SvgIcon';
/* eslint-disable max-len */

const Networkadress = props => (
  <SvgIcon {...props}>
    <svg
      focusable="false"
      xmlns="http://www.w3.org/2000/svg"
      viewBox="0 0 17.37 15"
      width="24px"
      height="24px"
    >
      <g id="Layer_2" data-name="Layer 2">
        <g id="Layer_1-2" data-name="Layer 1">
          <path d="M16.69,0H.68A.67.67,0,0,0,0,.68V11.83a.67.67,0,0,0,.68.67H4v1.82a.67.67,0,0,0,.68.68h8.09a.67.67,0,0,0,.68-.68V12.5h3.28a.68.68,0,0,0,.68-.67V.68A.68.68,0,0,0,16.69,0ZM16,11.15H12.73a.67.67,0,0,0-.67.68v1.82H5.31V11.83a.67.67,0,0,0-.67-.68H1.35V1.35H4v5H5.39v-5H6.68v5H8v-5H9.33v5h1.35v-5H12v5h1.35v-5H16Z" />
        </g>
      </g>
    </svg>
  </SvgIcon>
);

Networkadress.propTypes = {
  color: PropTypes.string,
};

Networkadress.defaultProps = {
  color: '#6d6d71',
};

export default Networkadress;
