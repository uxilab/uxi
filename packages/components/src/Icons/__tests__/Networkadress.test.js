import React from 'react';
import Networkadress from '../Networkadress';

describe('Icon : <Networkadress />', () => {
  it('match snapshot', () => {
    expect(shallow(<Networkadress />)).toMatchSnapshot();
  });
});

