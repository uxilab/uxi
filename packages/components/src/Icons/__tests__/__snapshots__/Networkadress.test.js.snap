// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon : <Networkadress /> match snapshot 1`] = `
<SvgIcon
  color="#6d6d71"
  hoverColor="#6d6d71"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  size={24}
  style={Object {}}
  viewBox="0 0 24 24"
>
  <svg
    focusable="false"
    height="24px"
    viewBox="0 0 17.37 15"
    width="24px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      data-name="Layer 2"
      id="Layer_2"
    >
      <g
        data-name="Layer 1"
        id="Layer_1-2"
      >
        <path
          d="M16.69,0H.68A.67.67,0,0,0,0,.68V11.83a.67.67,0,0,0,.68.67H4v1.82a.67.67,0,0,0,.68.68h8.09a.67.67,0,0,0,.68-.68V12.5h3.28a.68.68,0,0,0,.68-.67V.68A.68.68,0,0,0,16.69,0ZM16,11.15H12.73a.67.67,0,0,0-.67.68v1.82H5.31V11.83a.67.67,0,0,0-.67-.68H1.35V1.35H4v5H5.39v-5H6.68v5H8v-5H9.33v5h1.35v-5H12v5h1.35v-5H16Z"
        />
      </g>
    </g>
  </svg>
</SvgIcon>
`;
